themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -78,8 +78,15 @@ <h2> ${_("Student List Management")}</h2>
           </div>
         %endif
         <table>
+          <colgroup>
+            <col width="50">
+            <col width="70">
+            <col width="50">
+            <col width="35">
```

Comment:
please set `<col width="1" />` to shrink by content

Reply:
It doesn't work, 
![screenshot from 2018-11-01 17-13-07](https://user-images.githubusercontent.com/25189966/47843108-961edb80-ddf9-11e8-8994-90e32b935baf.png)